### ISSUE TITLE
GGMAX BUG WATCHTOWER & RAIN 

### ISSUE THREAD

**Issue by imothep85** [TheGameCreators/GameGuruRepo#5575]
When the player  is inside a watchtower, or using the stairs the RAIN disappear from the level.
![watchtower01](https://github.com/TheGameCreators/GameGuruRepo/assets/8946215/9d61e7e4-0c48-4984-80b4-e020910b8499)
![watchtower02](https://github.com/TheGameCreators/GameGuruRepo/assets/8946215/ece919d2-2038-4f1b-8da5-5bda03291ecd)
![watchtower03](https://github.com/TheGameCreators/GameGuruRepo/assets/8946215/c8ea49e2-5ad9-4171-9f14-d4e985d4e381)

**Comment by synchromesh62** [comment]
Not a bug that's just the way it works
When you go indoors the rain is disabled.
Fir now its just better than it raining indoors.

**Comment by MonkeyFrogStudio** [comment]
Yep, it's not a bug. However, other engines do have a means of blocking the rain indoors while it still rains outdoor. That way, if you're in a tower (like in the example here) or looking out a window in a building, you can see it's raining outside.

I know that S2 Engine, for example, has a blocker you can place around structures to keep it from raining there.

So, not a but, but it could be a request. ;)
